js-miniapp-bridge/src/types/fetch.ts

```diff
@@ -0,0 +1,4 @@
+export type MiniAppRequestInit = Pick<
+  RequestInit,
```

Comment:
I agree with @minh-rakuten about this - We should explicitly specify our own type rather than relying on part of the browser's/typescript's implementation. We are using the browser's `fetch` as inspiration, but there will end up being some differences.